examples/docs/en-US/table.md

```diff
@@ -1349,19 +1349,19 @@ Customize table column so it can be integrated with other components.
 ### Expandable row
 
 When the row content is too long and you do not want to display the horizontal scroll bar, you can use the expandable row feature.
-:::demo Activate expandable row by adding type="expand" and `inline-template` attribute，The template for `el-table-column` will be rendered as the contents of the expanded row, you can access the same attributes as the` inline-template`。
+:::demo Activate expandable row by adding type="expand" and `Scoped slot`, The template for `el-table-column` will be rendered as the contents of the expanded row, you can access the same attributes as the` Scoped slot`.
```

Comment:
Activate expandable row by adding type="expand" and `scoped slot`. The template for `el-table-column` will be rendered as the contents of the expanded row, and you can access the same attributes as when you are using`scoped slot` in custom column templates.